components/server/src/authorization/relationship-updater.ts

```diff
@@ -97,9 +97,6 @@ export class RelationshipUpdater {
                 });
                 return user;
             });
-        } catch (error) {
```

Comment:
we would just return the non migrated user before, which could cause strange concurrency effects. We now error if no migrated user can be returned.